tests/components/ct-react-vite/src/components/EmptyFragment.tsx

```diff
@@ -0,0 +1,3 @@
+export default function EmptyFragment() {
+  return <>{[]}</>;
```

Comment:
Note that this is not only about fragments from my understanding. In theory it's the same for:

- null
- undefined
- Fragments

Since just nothing gets rendered to the root element.